data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,947 @@
+			`	From above, this city has almost perfect radial symmetry, a sure sign that it was planned and built as a single unit rather than expanding at random over time. Every single building in the city is made from the same metal alloy, a dull and slightly irregular grey color. The corners of each building are slightly beveled instead of meeting at right angles.`
+			`	Even the spaceport is symmetrical: a ring of hangars, facing outwards, at evenly spaced intervals, with warehouses and living quarters inside the ring.`
+			`	While the layout is utilitarian, it is an incredibly comfortable place to live, and the Hai have given their individual community spaces quite a lot of character and life. It is, overall, an excellent example of an idealised Hai world, built in the presence of abundant resources.`
+			`	As you are wandering around and getting a feel for the place you overhear someone talking about an apparently complete ringworld in Alta Hai. It definitely sounds like something you should check out.`
+				accept
+
+	#Goto Stop 3 Wah Yoot (Darkcloak) - must happen after Quarg first contact
+mission "ZZ Hai Tour [3] Meet Unfettered"
+	landing
+	name "Meet the Rebels on <planet>"
+	description "Go to <destination> and learn more about the Hai's conflict."
+	source "Alta Hai"
+	destination "Darkcloak"
+	to offer
+		has "Hai Tour [2] Alta Hai: done"
+	on offer
+		conversation
+			`This is a complete Quarg ringworld, the product of millennia of construction effort which has consumed every other stellar object in this system in order to provide the raw materials. The inner edge of the ring is filled with solar panels, hydroponic gardens, living quarters, and sunlit common spaces. The outer edge contains hangars and docks for ships, and storage facilities for cargo.`
+			`	Pondering your questions, you find a Quarg to query, "So this isn't Hai space then, but it's a dead end?"`
+			`	The Quarg explains, "Much like in Enif in human space, this is where we live as friends to the Hai."`
+			`	This explanation seems typically Quarg, but it's the best one you're able to get.`
+			`	Before you go you ask, "Do you know about the rebels I've been hearing about?"`
+			`	The Quarg responds, "The Hai and their rebels are the same. You should travel to Darkcloak in Wah Yoot to understand."`
+			`	It seems like there's nothing else for it but to see for yourself.`
+				accept
+	
+	#Goto Stop 4 Wah Ki (Cloudfire)
+	# Must happen before unfettered first contact
+mission "00 Hai Tour [4] Cloudfire"
+	landing
+	name "Find out about the Rebels on Cloudfire"
+	description "Go to Cloudfire and compare the Hai's side of the conflict."
+	source "Darkcloak" 
+	destination "Cloudfire"
+	to offer
+		or
+			has "Hai Witness Unfettered [1]"
+			has "ZZ Hai Tour [3] Meet Unfettered: done"
+	on offer
+		conversation
+			`You had to bribe your way through just to land without being blown up. These Unfettered Hai do not seem friendly, but it's time to find out what they're about.`
```

Comment:
You might consider adding a sentence somewhere clarifying that the player should go to Cloudfire next.

Reply:
This mission only occurs if you haven't got Unfettered first contact. In which case the first contact mission sends you to cloudfire quite smoothly in this scenario.

Reply:
This is complex and it's been a while since I had a grasp on how everything flowed, but as long as the player is explicitly or very implicitly told to go to specifically Cloudfire, then it should be good.